lib-release.js

```diff
@@ -297,28 +297,28 @@ const confirmedRelease = (type, version) => {
           Step 1 complete...
         `));
     }),
+    () => execaSeries([
+      type === UNSTABLE ? mapCommands[`bump-${version === BETA ? '' : `${version}-`}beta`] : mapCommands[`bump-${version}`],
```

Comment:
@LucaMele 
Now the only possible issue I can think of is that if the `build` itself fails, we would have already a new version commited to `package.json` and a git tag would have been generated, which needs to be cleaned up upon error 🤔

Reply:
i guess yes..

Reply:
Okay I just checked the docs, and we could [prevent git tags and commits](https://www.npmjs.com/package/npmversion#possible-options), by:
```sh
--nogit-commit
        No git commit

    --nogit-tag
        No git tag
```